mf2py/dom_helpers.py

```diff
@@ -130,43 +131,58 @@ def text_collection(el, replace_img=False, img_to_src=True, base_url=''):
     results = [t for t in text_collection(el, replace_img, img_to_src, base_url) if t is not '']
 
     if results:
-        # remove <space> if it is first and last or if it is preceded by a <space> or <int> or followed by a <int>
+        # remove <space> if it is first and last or if it is preceded by a <space> or <p> open/close or followed by a <p> open/close
```

Comment:
one of those should read **pre** open/close?

Reply:
thanks! actually just needed to be reverted entirely. done.